.github/workflows/helm-multi-release.yaml

```diff
@@ -0,0 +1,199 @@
+name: Helm Multi Release
+# Reusable workflow for packaging and publishing Helm charts having multiple subcharts on GitHub pages
+
+on:
+  workflow_call:
+    inputs:
+      charts-dir:
+        description: "The directory containing the Helm chart and `.bumpversion.cfg` file."
+        required: true
+        type: string
+      root-branch:
```

Comment:
You can just use the variable GitHub provides by default instead of asking the user for it